tests/lib/rules/getter-return.js

```diff
@@ -0,0 +1,97 @@
+/**
+ * @fileoverview Enforces that a return statement is present in property getters.
+ * @author Aladdin-ADD(hh_2013@foxmail.com)
+ */
+
+"use strict";
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+const rule = require("../../../lib/rules/getter-return");
+const RuleTester = require("../../../lib/testers/rule-tester");
+
+//------------------------------------------------------------------------------
+// Tests
+//------------------------------------------------------------------------------
+
+const ruleTester = new RuleTester();
+
+// data is not working, so specify a name: "getter 'bar'"
+const name = "getter 'bar'";
+const noReturnMessage = `Expected to return a value in ${name}.`;
+const noLastReturnMessage = `Expected to return a value at the end of ${name}.`;
+const parserOptions = { ecmaVersion: 6 };
```

Comment:
Nitpick: Instead of using `parserOptions` for each test, you can set defaults:

```js
const ruleTester = new RuleTester({ parserOptions: { ecmaVersion: 6 } });
```

Then you won't need to repeat `parserOptions` in so many places.